app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserListAdapter.java

```diff
@@ -604,60 +609,10 @@ public int getFilePosition(final File file) {
     private static final ExecutorService executorService = new ThreadPoolExecutor(0, 3, 60, TimeUnit.SECONDS, new SynchronousQueue<>());
 
     private void loadFolder(final File folder) {
-        executorService.execute(new FolderLoader(folder));
```

Comment:
I moved all the code from the FolderLoader back in to the loadFolder function. This is not strictly necessary - I just thought it was cleaner.